testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -66,6 +85,11 @@ def test_load():
         assert_equal(len(u.atoms), 3341, "Loading universe failed somehow")
 
     @staticmethod
+    def test_load_stringio():
+        u = mda.Universe(StringIO(CHOL_GRO), format='GRO')
```

Comment:
I'd still like this to test that it has been loaded with a trajectory, so something simple like checking a position of an atom